### ISSUE TITLE
CVar chatBubbles and chatBubblesParty might prevent Interface in Setting Panel to load

### ISSUE THREAD

**Issue by LiangYuxuan** [Stanzilla/WoWUIBugs#289]
Certain CVar values will let Setting Panel's Interface tab failing to load by rasing error due to failing type check.

Reproduced on 10.0.2.45779.

1. Set CVar values `/run C_CVar.SetCVar('chatBubbles', '0'); C_CVar.SetCVar('chatBubblesParty', '1')`
2. Reload UI
3. UI loaded with lua error, and Interface tab won't load

###### Error log
```
Message: Interface/SharedXML/Settings/Blizzard_Setting.lua:221: SetValue '聊天泡泡' requires 'number' type, not 'nil' type.
Time: Sat Oct  1 00:05:35 2022
Count: 1
Stack: Interface/SharedXML/Settings/Blizzard_Setting.lua:221: SetValue '聊天泡泡' requires 'number' type, not 'nil' type.
[string "=[C]"]: in function `error'
[string "@Interface/SharedXML/Settings/Blizzard_Setting.lua"]:221: in function `SetValueInternal'
[string "@Interface/SharedXML/Settings/Blizzard_Setting.lua"]:237: in function `Init'
[string "@Interface/SharedXML/Mixin.lua"]:43: in function `CreateAndInitFromMixin'
[string "@Interface/SharedXML/Settings/Blizzard_Settings.lua"]:181: in function `RegisterProxySetting'
[string "@Interface/FrameXML/SettingDefinitions/Interface.lua"]:326: in function `registrant'
[string "@Interface/SharedXML/Settings/Blizzard_SettingsRegistrar.lua"]:15: in function <...ce/SharedXML/Settings/Blizzard_SettingsRegistrar.lua:14>
[string "=[C]"]: in function `secureexecuterange'
[string "@Interface/SharedXML/Settings/Blizzard_SettingsRegistrar.lua"]:17: in function <...ce/SharedXML/Settings/Blizzard_SettingsRegistrar.lua:11>
[string "@Interface/SharedXML/EventUtil.lua"]:21: in function <Interface/SharedXML/EventUtil.lua:15>
[string "=[C]"]: ?
[string "@Interface/SharedXML/CallbackRegistry.lua"]:143: in function <Interface/SharedXML/CallbackRegistry.lua:142>
[string "=[C]"]: ?
[string "@Interface/SharedXML/CallbackRegistry.lua"]:146: in function `TriggerEvent'
[string "@Interface/SharedXML/GlobalCallbackRegistry.lua"]:9: in function <Interface/SharedXML/GlobalCallbackRegistry.lua:8>

Locals: (*temporary) = "SetValue '聊天泡泡' requires 'number' type, not 'nil' type."
```

**Comment by Meorawr** [comment]
Should be fixed as of build 10.0.2.45969.

**Comment by Meorawr** [comment]
Going to re-open this and see if we can get it fixed in the 10.0.0 PTR too as it's unresolved there.
